### PR TITLE
Alcoholic Additives

### PR DESCRIPTION
**Pull request by Sylrene**
Adds new mixed drinks and options to the alcohol dispensers.

### CODE REVIEW

File: code/modules/reagents/dispenser/dispenser_presets.dm [hunk 1]
```diff
@@ -118,6 +118,10 @@
 			/obj/item/weapon/reagent_containers/chem_disp_cartridge/cognac,
 			/obj/item/weapon/reagent_containers/chem_disp_cartridge/ale,
 			/obj/item/weapon/reagent_containers/chem_disp_cartridge/mead
+			/obj/item/weapon/reagent_containers/chem_disp_cartridge/mead,
```
Comment (io-util) [inline code/modules/reagents/dispenser/dispenser_presets.dm]: This should not be here 2 times, the first is missing a comma too. Remove one of them and make sure the one that stays has a comma, else the list will break.

**Review by io-util — CHANGES_REQUESTED** [review]

**Review by Jackofashepard — APPROVED** [review]

File: code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Food-Drinks.dm [hunk 2]
```diff
@@ -3002,3 +3002,142 @@
...
+	taste_description = "rustic chocolate"
+	color = "#2e2312"
+	strength = 50
+
+	glass_name = "creme de cacao"
+	glass_desc = "Perfect for a night in the lodge."
+
+/datum/reagent/ethanol/creme_de_menthe
+	name = "Creme de Menthe"
+	description = "A sweet, mint-flavored alcoholic beverage"
```
Comment (EvenInDeathIStillServe) [inline code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Food-Drinks.dm]:
```suggestion
	description = "A sweet, mint-flavored alcoholic beverage."
```

File: code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Food-Drinks.dm [hunk 3]
```diff
@@ -3002,3 +3002,142 @@
...
+	glass_desc = "Make sure to set it down after drinking."
+
+/datum/reagent/ethanol/between_the_sheets/affect_ingest(mob/living/carbon/M, alien, removed)
+	. = ..()
+	var/strength_mod = 1
+	if(alien == IS_DIONA)
+		return
+	if(alien == IS_SKRELL)
+		strength_mod *= 2
+	if(prob(1*strength_mod))
```
Comment (EvenInDeathIStillServe) [inline code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Food-Drinks.dm]:
```suggestion
	if(prob(strength_mod))
```
1*something = something

File: code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Food-Drinks.dm [hunk 4]
```diff
@@ -3002,3 +3002,142 @@
...
+
+/datum/reagent/ethanol/stinger
+	name = "Stinger"
+	description = "A popular drink, known for its ability to mask the taste of inferior brandies with the strong flavor of creme de menthe."
+	taste_description = "the prohibition"
+	color = "#8f8465"
+	strength = 13
+
+	glass_name = "Stinger"
+	glass_desc = "Holding this glass will make you seem like high-soceity... if you were from the 1800s."
```
Comment (EvenInDeathIStillServe) [inline code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Food-Drinks.dm]:
```suggestion
	glass_desc = "Holding this glass will make you seem like high-society... if you were from the 1800s."
```

**Review by EvenInDeathIStillServe — CHANGES_REQUESTED** [review]
Just a couple grammar mistakes, and an unnecessary multiplication.

**Review by io-util — CHANGES_REQUESTED** [review]
Fix the small stuff suggested above and it's a go from me. @Sylrene

File: code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Food-Drinks.dm (hunk 4 above)
Comment (Mojamble) [reply code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Food-Drinks.dm]: we live in a soceity

Also, I should have gone over this myself before handing it over to Sy. This is on me.

**Review by io-util — APPROVED** [review]
Looks okay now.

**Pull request merged**